component/Lists/list.html

```diff
@@ -54,20 +54,20 @@ <h3 class="text-lg text-light">Lists </h3>
                     <span class="text-xm ">Spaced Lists</span>
                 </div>
                 <div class=" box-shadow-2 m-auto wt-50">
-                    <ul class="flex justify-around ">
-                        <li class="style-none pd-3 ">
+                    <ul class="flex justify-around style-none">
+                        <li class=" pd-3 ">
                             List 1
                         </li>
-                        <li class="style-none pd-3 ">
```

Comment:
instead of every syle none in list element you could also make one modifier class for this particular div of list , it make goodon web performence

Reply:
Yes I have changed.

Reply:
good